recipes/hlama/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: hlama
+  version: "0.2"
+
+build:
+  number: 0
+  skip: True # [py27]
+  entry_points:
+    - hlama = hlama.app:main
+
+source:
+  fn: hlama-develop.tar.gz
+  url: https://github.com/bihealth/hlama/archive/develop.tar.gz
```

Comment:
Here is a checksum missing and it would be nice to track a tagged version or a specific commit. Moreover we need a rebase.